Dockerfile

```
@@ -23,17 +23,17 @@ RUN ./gradlew assemble
 
 
 # ---
-FROM eclipse-temurin:17.0.6_10-jre-alpine AS final
+FROM eclipse-temurin:19-jre-jammy AS final
 LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
 
 # force a rebuild of `apk upgrade` below by invalidating the BUILD_NUMBER env variable on every commit
 ARG BUILD_NUMBER
 ENV BUILD_NUMBER ${BUILD_NUMBER:-1_0_0}
 
-RUN apk upgrade --no-cache && \
-     apk add --no-cache \
-       curl \
-       tzdata
+RUN apt-get update && \
+    apt-get -y upgrade && \
+    apt-get install -y curl && \
+    apt-get install -y tzdata
```

```suggestion
    apt-get install -y \
      curl \
      tzdata \
    && rm -rf /var/lib/apt/lists/*
```
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get

> Below is a well-formed RUN instruction that demonstrates all the apt-get recommendations.
>
>     RUN apt-get update && apt-get install -y \
>         aufs-tools \
>         automake \
>         build-essential \
>         curl \
>         dpkg-sig \
>         libcap-dev \
>         libsqlite3-dev \
>         mercurial \
>         reprepro \
>         ruby1.9.1 \
>         ruby1.9.1-dev \
>         s3cmd=1.1.* \
>      && rm -rf /var/lib/apt/lists/*

---

Thanks David, I will update it accordingly